docs/StardustDocs/cfg/buildprofiles.xml

```diff
@@ -17,6 +17,7 @@
         <enable-browser-edits>true</enable-browser-edits>
         <browser-edits-url>https://github.com/Kotlin/dataframe/edit/master/docs/StardustDocs/</browser-edits-url>
     </variables>
+    <sitemap priority="0.35" change-frequency="monthly"/>
```

Comment:
When do you define the priority number, how do you generate these numbers?

Reply:
Nope, just what the docs says https://www.jetbrains.com/help/writerside/generate-sitemap.html. To be honest, I'm not sure what it does.

Reply:
I could probably just leave out the priority, since it's optional